src/commissaire/models/__init__.py

```diff
@@ -168,28 +168,40 @@ def _dict_for_json(self, secure):
                 data[key] = getattr(self, key)
         return data
 
-    def to_json(self):
+    def to_json(self, expose=None):
         """
         Returns a JSON representation of this model.
 
+        :param expose: List of non exposed attributes to include in result.
+        :type expose: list
         :returns: The JSON representation.
         :rtype: str
         """
-        return json.dumps(
-            self._struct_for_json(secure=True),
-            default=lambda o: o._struct_for_json(secure=True))
+        struct = self._struct_for_json(secure=True)
+        if expose:
```

Comment:
I was thinking about [this](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments) but since we are not appending I think you are right.